.golangci.yml

```diff
@@ -1,19 +1,33 @@
 run:
   timeout: 5m
+  go: "1.19"
   skip-files:
     - "zz_generated_*"
   skip-dirs:
     - pkg/generated
   tests: false
+  allow-parallel-runners: true
+
 output:
   format: github-actions
+
 linters:
+  disable-all: true
```

Comment:
As a personal preference i much prefer enable all and then explicitly disabling linters.

Reply:
While I agree with your point, I found `enable-all: true ` be disruptive at times, because under the hood it means: whenever a new linter is added or upgraded upstream, it will try to enable it thus making the probability of build failures high and somewhat uncontrollable, whereas `disable-all: true` and providing the specific linters we want to use to `enable:` list is much more controlled.